cns/restserver/ipam.go

```diff
@@ -434,26 +563,46 @@ func (service *HTTPRestService) unassignIPConfig(ipconfig cns.IPConfigurationSta
 func (service *HTTPRestService) releaseIPConfig(podInfo cns.PodInfo) error {
```

Comment:
the func name needs to be updated to reflect it may be releasing multiple IP configs.
need releaseIPConfigs